### ISSUE TITLE
[Accessibility][Sourcing] high contrast issues for fd-radio button and fd-popover

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9082]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
1) Unable to view the selected radio button in high contrast mode
2) Border is not visible for fd-popover in high contrast mode

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
windows 10 os
Angular 13
   "@fundamental-ngx/core": "0.35.4",
    "@fundamental-ngx/platform": "0.35.4",

#### If this is a bug, please provide steps for reproducing it.
 Unable to view the selected radio button in high contrast mode
1) Access the url https://sap.github.io/fundamental-ngx/fundamental-ngx#/core/radio 
and turn on high contrast mode on windows 10 machine
2) Click on any radio button 
3) Notice that radio button selection mode is not visible - please refer video recording attached

Border is not visible for fd-popover in high contrast mode
1) Access https://sap.github.io/fundamental-ngx/fundamental-ngx#/core/popover open a popover 
2) Turn on high contrast mode on windows 10 machine 
3) Notice that popover border is not visible - please refer to attached video recording

#### Please provide relevant source code if applicable.
Please refer to source code in https://sap.github.io/fundamental-ngx/fundamental-ngx#/core/popover and https://sap.github.io/fundamental-ngx/fundamental-ngx#/core/radio

#### Is there anything else we should know?
This is a blocker for Guided sourcing customer and request you to please take it as high priority

https://user-images.githubusercontent.com/88182358/206997564-2760b577-52f3-4e51-8b8c-cac8ccc75b4f.mp4

https://user-images.githubusercontent.com/88182358/206997585-43a1c4ff-2567-4cb9-8cca-75acf0e9f020.mp4

**Comment by droshev** [comment]
It should be fixed with the latest Fundamental Styles and NGX

**Comment by I543348** [comment]
This is still reproducible , please reopen the issue
  "@fundamental-ngx/core": "0.39.0-rc.66",
    "@fundamental-ngx/platform": "0.39.0-rc.66",

<img width="380" alt="Task_40837_Fail" src="https://user-images.githubusercontent.com/88182358/223004792-d72af343-17b7-441a-8d35-41046d48c740.png">

**Comment by droshev** [comment]
I am not able to reproduce it
<img width="1774" alt="Screenshot 2023-03-06 at 5 34 33 PM" src="https://user-images.githubusercontent.com/4380815/223265355-4a40927c-ab2d-408e-9bbb-b6e8a99b47a0.png">
<img width="1706" alt="Screenshot 2023-03-06 at 5 35 11 PM" src="https://user-images.githubusercontent.com/4380815/223265365-a166e28a-7a2d-4276-a778-aac302e6e598.png">
<img width="1695" alt="Screenshot 2023-03-06 at 5 34 54 PM" src="https://user-images.githubusercontent.com/4380815/223265368-16f1206e-fdef-411f-8dfb-01ee275a15cd.png">

**Comment by I543348** [comment]
Hi @droshev , the issue is not reported for "High contrast black theme" but rather windows OS high contrast, the end users or customers will use the OS high contrast mode, the screenshot attached is from windows machine.

Please check and confirm.

**Comment by I543348** [comment]
Hi @mikerodonnell89 , 
the css for detecting os high contrast mode using media queries is as below -

@media screen and (-ms-high-contrast: active), screen and (forced-colors: active) {
// your styles here
  }

**Comment by I543348** [comment]
This needs to be reopened 
cc: @droshev , @mikerodonnell89

**Comment by droshev** [comment]
> This needs to be reopened as high contrast mode itself is gone now for fd-popover cc: @droshev , @mikerodonnell89

@I543348 Can you describe the problem you are facing?

**Comment by I543348** [comment]
Sure @droshev ,
Guided sourcing application doesn't support theming currently,  hence disabled users are using OS high contrast feature.
The style changes from NGX after fix is not working because when the user turns on OS high contrast mode, the fd-popover is coming in normal color as in screenshot https://github.com/SAP/fundamental-ngx/issues/9082#issuecomment-1455330724 rather than in black or white color (high contrast mode)

hence , we are looking for the changes done on F-NGX side media queries so that we can revert the changes and provide fix from application side , because before NGX fix , only the fd-popover borders were broken but now looks like fd-popover body and header comes in normal color even in high contrast mode.
 Please let me know if any further information is needed.

**Comment by droshev** [comment]
If you are using Fundamental NGX you have options of using all the themes. The recommendation is if you are in Windows High Contrast to switch to HCB. If you do so then the popover will look 
<img width="493" alt="Screenshot 2023-04-04 at 12 47 04 PM" src="https://user-images.githubusercontent.com/4380815/229861627-bf7118dc-6328-4516-b36b-754f6ef91daa.png">

**Comment by I543348** [comment]
Hi @droshev , yes thats true , currently guided sourcing application does not support theme options for end users
hence customers use high contrast mode from native OS only, so we want to fix it from application side and want to revert changes from NGX side in application.

That's the reason , we want to know what changes are done on media queries from NGX for this issue

**Comment by droshev** [comment]
@I543348 the non-high contrast themes(Morning Horizon or Evening Horizon) don't contain anything specific for high contrast. There isn't any "fix" there and we don't plan any "fix".

**Comment by I543348** [comment]
Thanks @droshev , but what i understand is before this issue was logged fd-popover was working fine but with a broken border, after the changes and after the comment https://github.com/SAP/fundamental-ngx/issues/9082#issuecomment-1371183357 , its now not working as earlier in windows HCB mode. Hence we wanted to understand on the changes done after the comment. 
Could you please confirm the same? It would be really helpful.

**Comment by droshev** [comment]
> Thanks @droshev , but what i understand is before this issue was logged fd-popover was working fine even in windows HCB but with a broken border, after the changes and after the comment [#9082 (comment)](https://github.com/SAP/fundamental-ngx/issues/9082#issuecomment-1371183357) , its now not working as earlier in windows HCB mode. Hence we wanted to understand on the changes done after the comment. Could you please confirm the same? It would be really helpful.

Unfortunately, we can't help you in that request.
